pkg/pillar/dpcreconciler/linux.go

```diff
@@ -1406,23 +1444,92 @@ func (r *LinuxDpcReconciler) getIntendedWwanConfig(dpc types.DevicePortConfig,
 			physAddress.Interface = port.IfName
 		}
 		network := types.WwanNetworkConfig{
-			// TODO: Username + Password (will be done in the next PR)
-			LogicalLabel:     port.Logicallabel,
-			PhysAddrs:        physAddress,
-			SIMSlot:          accessPoint.SIMSlot,
-			APN:              accessPoint.APN,
-			PreferredPLMNs:   accessPoint.PreferredPLMNs,
-			PreferredRATs:    accessPoint.PreferredRATs,
-			ForbidRoaming:    accessPoint.ForbidRoaming,
-			Proxies:          port.Proxies,
-			Probe:            probeCfg,
-			LocationTracking: locationTracking,
+			LogicalLabel:      port.Logicallabel,
+			PhysAddrs:         physAddress,
+			SIMSlot:           accessPoint.SIMSlot,
+			APN:               accessPoint.APN,
+			AuthProtocol:      accessPoint.AuthProtocol,
+			Username:          decBlock.CellularNetUsername,
+			EncryptedPassword: encPassword,
```

Comment:
I don't understand what detsecrets/yetus is complaining about here.

Reply:
Neither do I.
I have at least fixed the other Yetus complains.

Reply:
I wonder if it is the name of the field in the struct that makes the tool think something odd is going on.

Reply:
Yes, seems to be the case: https://github.com/Yelp/detect-secrets/blob/master/detect_secrets/plugins/keyword.py#L56

Reply:
Hmm - we should really disable this detsecrets since it triggers on this but doesn't appear to look for apikeys and other more common mistakes. But let's wory about that later.